addons/web/static/src/js/views/search_menus.js

```diff
@@ -231,6 +232,23 @@ return Widget.extend({
         }
         this.query.reset([this.facet_for(filter)], {
             preventSearch: preventSearch || false});
+
+        // Load sort settings on view
+        var self = this;
+        var sort_items = JSON.parse(filter.sort);
+        if (sort_items) {
+            _.each(sort_items.reverse(), function(sort_order) {
```

Comment:
instead of this all split, wouldn't `dataset.set_sort(sort_items)` do the trick ?